File: app/src/main/java/edu/itschool/abitpro/activity_search.java
package edu.itschool.abitpro;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import edu.itschool.abitpro.databinding.ActivitySearchBinding;

public class activity_search extends AppCompatActivity {

    private ActivitySearchBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivitySearchBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/edu/itschool/abitpro/activity_search.kt
package edu.itschool.abitpro

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.ui.AppBarConfiguration
import edu.itschool.abitpro.databinding.ActivitySearchBinding

class activity_search : AppCompatActivity() {
    private val appBarConfiguration: AppBarConfiguration? = null
    private var binding: ActivitySearchBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivitySearchBinding.inflate(getLayoutInflater())
    }
}